sockeye/model.py

```diff
@@ -11,70 +11,50 @@
-                                                      "conv_embed_max_filter_width",
-                                                      "conv_embed_num_filters",
-                                                      "conv_embed_pool_stride",
-                                                      "conv_embed_num_highway_layers",
-                                                      "rnn_cell_type",
-                                                      "rnn_num_layers",
-                                                      "rnn_num_hidden",
-                                                      "rnn_residual_connections",
-                                                      "weight_tying",
-                                                      "context_gating",
-                                                      "lexical_bias",
-                                                      "learn_lexical_bias",
-                                                      "data_info",
-                                                      "loss",
-                                                      "normalize_loss",
-                                                      "smoothed_cross_entropy_alpha",
-                                                      "layer_normalization",
-                                                     ],
-                                                     default_values={
-                                                      "attention_use_prev_word": False,
-                                                      "context_gating": False,
-                                                      "loss": C.CROSS_ENTROPY,
-                                                      "normalize_loss": False,
-                                                      "layer_normalization": False,
-                                                      "encoder": C.RNN_NAME,
-                                                      "conv_embed_max_filter_width": 8,
-                                                      "conv_embed_num_filters": None,
-                                                      "conv_embed_pool_stride": 5,
-                                                      "conv_embed_num_highway_layers": 4,
-                                                     })
-"""
-ModelConfig defines model parameters defined at training time which are relevant to model inference.
-Add new model parameters here. If you want backwards compatibility for models trained with code that did not
-contain these parameters, provide a reasonable default under default_values.
-"""
+
+class ModelConfig(Config):
+    """
+    ModelConfig defines model parameters defined at training time which are relevant to model inference.
+    Add new model parameters here. If you want backwards compatibility for models trained with code that did not
+    contain these parameters, provide a reasonable default under default_values.
```

Comment:
default_values don't seem to exist anymore. How does one do backwards compatible changes in the future? Can you just add arguments with default values to the config classes?